metadata.json

```diff
@@ -3,7 +3,7 @@
   "version": "2.0.2",
   "author": "puppetlabs",
   "summary": "Provides Puppet types and providers to manage Logical Resource Management (LVM) features.",
-  "license": "GPL-2.0",
```

Comment:
are you sure you're allowed to switch the license? Did all contributors sign a CLA?

Reply:
AFAIK this module will be maintain by CAT (Content and Tooling) team and all other modules maintain by Apache license only, that's is where we have decided to go with updating license from GPL to Apache.

Reply:
Yeah fine but you cant just change it to whatever you like, I'm quite sure the GPL doesn't allow it.